scripts/poll.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const votingButton = document.querySelector('#vote-button');
```

Comment:
@ange-stephane kindly consider adding and using class along with [querySelectorAll](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll) to fetch all vote buttons. Using querySelector will only fetch the first match and ignore the rest.
`const votingButtons = document.querySelectorAll('.vote-button');`
Once you fetch all the button, you can loop through and dynamically add event listener to each button using event propagation technique.
`votingButtons.forEach(button => {
      button.addEventListener('click', function(event) {
          console.log(event);
      })
})`